figures/tasks.py

```diff
@@ -328,6 +374,63 @@ def populate_daily_metrics_next(site_id=None, force_update=False):
                            site_count=sites_count))
 
 
+@shared_task
+def daily_metrics_callback(results, *args):
```

Comment:
@OmarIthawi looks like just a logger function. Does it or will it serve any other purpose? If it is only for logging, perhaps add a logging context to the function name?